src/Avalonia.Visuals/Media/HsvColor.cs

```diff
@@ -0,0 +1,493 @@
+        }
+
+        /// <summary>
+        /// Converts the given RGB color to it's HSV color equivalent.
+        /// </summary>
+        /// <param name="color">The color in the RGB color model.</param>
+        /// <returns>A new <see cref="HsvColor"/> equivalent to the given RGBA values.</returns>
+        public static HsvColor FromRgb(Color color)
+        {
+            return HsvColor.FromRgb(color.R, color.G, color.B, color.A);
+        }
+
+        /// <summary>
+        /// Converts the given RGBA color channel values to it's HSV color equivalent.
+        /// </summary>
+        /// <param name="red">The red channel value in the RGB color model.</param>
+        /// <param name="green">The green channel value in the RGB color model.</param>
+        /// <param name="blue">The blue channel value in the RGB color model.</param>
+        /// <param name="alpha">The alpha channel value.</param>
+        /// <returns>A new <see cref="HsvColor"/> equivalent to the given RGBA values.</returns>
+        public static HsvColor FromRgb(
+            byte red,
+            byte green,
+            byte blue,
+            byte alpha = 0xFF)
+        {
+            // Note: Conversion code is originally based on the C++ in WinUI (licensed MIT)
+            // https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/Common/ColorConversion.cpp
+            // This was used because it is the best documented and likely most optimized for performance
+            // Alpha channel support was added
+
+            // Normalize RGBA channel values into the 0..1 range used by this algorithm
+            double r = red / 255.0;
+            double g = green / 255.0;
+            double b = blue / 255.0;
+            double a = alpha / 255.0;
+
+            double hue;
+            double saturation;
+            double value;
+
+            double max = r >= g ? (r >= b ? r : b) : (g >= b ? g : b);
+            double min = r <= g ? (r <= b ? r : b) : (g <= b ? g : b);
+
+            // The value, a number between 0 and 1, is the largest of R, G, and B (divided by 255).
+            // Conceptually speaking, it represents how much color is present.
+            // If at least one of R, G, B is 255, then there exists as much color as there can be.
+            // If RGB = (0, 0, 0), then there exists no color at all - a value of zero corresponds
+            // to black (i.e., the absence of any color).
+            value = max;
+
+            // The "chroma" of the color is a value directly proportional to the extent to which
+            // the color diverges from greyscale.  If, for example, we have RGB = (255, 255, 0),
+            // then the chroma is maximized - this is a pure yellow, no gray of any kind.
+            // On the other hand, if we have RGB = (128, 128, 128), then the chroma being zero
+            // implies that this color is pure greyscale, with no actual hue to be found.
+            double chroma = max - min;
```

Comment:
```suggestion
            var chroma = max - min;
```

Reply:
Same as above... this is pedantic and shouldn't be picked up by reviewers IMO.